build.gradle

```diff
@@ -16,3 +22,7 @@ android {
         consumerProguardFiles 'consumer-proguard.pro'
     }
 }
+
+dependencies {
+    api 'com.usebutton.merchant:button-merchant:1.0.0'
```

Comment:
I think we should use implementation @ecgreb let me know what you think about this

Reply:
@ecgreb Thoughts on this?

Reply:
I don't think we would want partners calling `configure()` again on a separate instance but it could still be powerful to give them access to the underlying instance that is powering the integration. Is there any way to do this?

Reply:
While we don't provide _direct_ access to the instance, we have wrapper methods that allow access to key merchant library API.

Reply:
On another note, would we want to use a wildcard versioning for only 1.x of the Merchant Library dependency? Since we only make API breaking changes on major versions. This'll allow the mParticle ButtonKit to get the latest of bug fixes for free without manually having to increment the version upon every Merchant Library update.